### ISSUE TITLE
Issue with tree-sitter-css

### ISSUE THREAD

**Issue by tientrinh21** [dracula/vim#288]
# Problem
Hello, I was trying to use treesitter these day, dracula works fine for me in HTML, JS and TS except CSS (with tree-sitter-css installed). All the class and properties appeared with no highlighting.

## What happened
![image](https://user-images.githubusercontent.com/57804513/183936665-28255e74-927c-4e02-988a-489016a2bf03.png)

## What I expected to happen
This is from neovim basic highlighting with  css3 syntax plugins ('hail2u/vim-css3-syntax')

![image](https://user-images.githubusercontent.com/57804513/183937509-cb604b83-541e-4c2c-8f7b-b253e3c7ef53.png)

## Machine Info
- **Vim type (`vim`/`gvim`/`neovim`)**: neovim
- **Vim version**: 0.7.2
- **OS**: Debian

**Comment by benknoble** [comment]
We're going to need more information to be helpful.

What is "tree-sitter-css"? And what highlight groups are applied to the unhighlighted portion? (You can see how to find out [here](https://vi.stackexchange.com/a/22105/10604) or [here](https://vi.stackexchange.com/a/11877/10604), for example.)

Can you share a minimum reproducible example? You might look at [How to debug my vimrc](https://vi.stackexchange.com/q/2003/10604) for tips on reducing your configuration to the minimum necessary to demonstrate the issue.

Since this is NeoVim, I'll ping @dsifford

**Comment by dsifford** [comment]
Yeah, I'm aware of this. We need someone to go in and take the time/effort to update the tree sitter groups specifically for css.

I've been meaning to get to it, but I so rarely use plain `.css` that it slips my mind.

PR encouraged if you feel up for it @tientrinh21

**Comment by benknoble** [comment]
Ah, seems I was missing some information. Thanks for clarifying Derek.

**Comment by tientrinh21** [comment]
Oh, thank you for all of your replies. I guess using another plugin could be a temporary alternative while figuring with tree-sitter groups for CSS.

**Comment by stnley** [comment]
@dsifford Is the intent to duplicate, as needed, the groups already defined in `after/syntax`?

Pic below, left side tree-sitter disabled, right side tree-sitter enabled.
![1698591528_grimshot](https://github.com/dracula/vim/assets/64174376/bb7cef11-c841-4b63-864a-83e4116884a0)

This is after `:hi! link @property.css cssProp`
![1698592158_grimshot](https://github.com/dracula/vim/assets/64174376/3f9e9683-d5a8-4056-a43e-5d363c442b49)

It looks to me like the built-in syntax is more specific than the CSS tree-sitter queries. Pretty much everything is just `@property.css`. Feels like this is something that requires upstream to improve.
